windicss/deps.ts

```diff
@@ -1,3 +1,11 @@
-export { default as Processor } from "https://esm.sh/windicss@3.1.2";
-export { HTMLParser } from "https://esm.sh/windicss@3.1.2/utils/parser";
-export { StyleSheet } from "https://esm.sh/windicss@3.1.2/utils/style";
+export { default as Processor } from "https://esm.sh/windicss@3.3.0";
+export {
+  CSSParser,
+  HTMLParser,
+} from "https://esm.sh/windicss@3.3.0/utils/parser";
+export { StyleSheet } from "https://esm.sh/windicss@3.3.0/utils/style";
+
+export {
+  Element,
+  HTMLDocument,
+} from "https://deno.land/x/deno_dom@v0.1.19-alpha/deno-dom-wasm.ts";
```

Comment:
this library is included in Lume, so you could import it directly with `lume/deps/dom.ts` in windicss.ts (this ensures that we are using the same version in both places)